### ISSUE TITLE
Update wiki to match with current functionality

### ISSUE THREAD

**Issue by jkubrynski** [gnab/remark#184]
Remark is growing - which is of course cool :) But there are a lot of features developed due to issues which are not in wiki or any other documentation. That's cool that we can specify slideNumberFormat function but since it's not documented it's used probably only by issue reporter and remark author :)

**Comment by gnab** [comment]
You are correct. The wiki is outdated. But the slideNumberFormat is actually mentioned [here](https://github.com/gnab/remark/wiki/Configuration), in the General section :)

Also, the wiki is open for editing, so if you find anything missing or in need of change, you're welcome to edit it yourself :)

**Comment by peterj** [comment]
@jkubrynski do you know of any other specific features that are not documented?
